marketplace/mux-video-uploader/extension.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "mux-video-uploader",
+  "name": "Mux Video Uploader",
+  "src": "mux-video-uploader-1.contentfulexts.com",
+  "fieldTypes": ["Object"],
```

Comment:
Why is this using the field type `Object`? As far as I could tell is stores a simple ID which could be a `Symbol`.

Reply:
I could be misunderstanding something, but as far as I can tell this is an object. The [`MuxContentfulObject` interface](https://github.com/contentful/extensions/pull/118/files#diff-5f6c0a600833ccc258e7933b210cc871R21) describes the field object.